src/platformdirs/api.py

```diff
@@ -5,12 +5,15 @@
 import os
 from abc import ABC, abstractmethod
 from pathlib import Path
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, final
 
 if TYPE_CHECKING:
     from collections.abc import Iterator
     from typing import Literal
 
+TEMP_ENV_VARS = "TMPDIR", "TEMPDIR", "TEMP", "TMP"
+"""A tuple of environment variables that can be used to override the cache directory."""
```

Comment:
In sphinx would the way to attach doc to variables is `#: ` on the right handside.

Reply:
Until there's a reuse generalizing this is a YAGNI. Inline it where you're using it, and use next with a comprehension instead of for loop.

Reply:
The tests should not use this variable, otherwise how do you catch when someone by mistake removes one of the values from it?

Reply:
I'll write a test for that since you ask, but I'm not worried about that; you'd easily catch it.

Reply:
The test suite should catch it, not the reviewer.